src/main/java/uk/gov/hmcts/ccd/data/casedataaccesscontrol/RoleAssignmentAttributesResource.java

```diff
@@ -16,9 +17,9 @@
 public class RoleAssignmentAttributesResource implements Serializable {
     private static final long serialVersionUID = -7106266789404292869L;
 
-    String jurisdiction;
-    String caseId;
-    String region;
-    String location;
-    String contractType;
+    Optional<String> jurisdiction;
```

Comment:
can we have an integration test that proves nothing breaks in our flow when new unknown attributes are returned by the API please

Reply:
added integration tests for unknown fields